pkg/cmd/bazci/githubpost/githubpost.go

```diff
@@ -78,14 +78,15 @@ func defaultFormatter(ctx context.Context, f failure) (issues.IssueFormatter, is
 		}
 	}
 	return issues.UnitTestFormatter, issues.PostRequest{
-		TestName:        f.testName,
-		PackageName:     f.packageName,
-		Message:         f.testMessage,
-		Artifacts:       "/", // best we can do for unit tests
-		HelpCommand:     issues.UnitTestHelpCommand(repro),
-		MentionOnCreate: mentions,
-		ProjectColumnID: projColID,
-		ExtraLabels:     extraLabels,
+		TestName:             f.testName,
+		PackageName:          f.packageName,
+		Message:              f.testMessage,
+		Artifacts:            "/", // best we can do for unit tests
+		HelpCommand:          issues.UnitTestHelpCommand(repro),
+		MentionOnCreate:      mentions,
+		ProjectColumnID:      projColID,
+		ExtraLabels:          extraLabels,
+		SkipLabelTestFailure: os.Getenv("NO_C_TEST_FAILURE") != "",
```

Comment:
While I don't think this needs a `COCKROACH_` prefix necessarily, I would probably choose an env var name that is closer to the post request option this affects: maybe `SKIP_LABEL_TEST_FAILURE`.

I also think it should be relatively straightforward to add a test case for this in `TestListFailuresFromJSON`.

@cockroachdb/dev-inf would know better if there's a place such env vars are documented; I'm not aware of them being listed anywhere.

Reply:
I made some incorrect assumptions about the test case. Unittest added and PR/commit updated to the right package!